_posts/2025-01-23-ad-rule-cx-success.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+
+title:  "Enhancing anomaly detection in Amazon OpenSearch Service: A customer success story"
+authors:
+ - kaituo
+ - kentonparton
+ - amitgalitz
+date: 2025-01-22
```

Comment:
2025-03-07

Reply:
@krisfreedain @kolchfa-aws - Can get your help with pushing this live today, pretty please.